docs/loadbalancing/loadbalancing-solution.md

```diff
@@ -30,21 +30,21 @@ For creating a highly available kubernetes cluster, Loadbalancers for kubeAPI se
 
 ### Role 
 
-The Claudie uses concept or roles while configuring the loadbalancers in input manifest. Each role represents a loadbalancer configuration for a particular use. Roles are then assigned to the Loadbalancer cluster. Single loadbalancer cluster can have multiple roles assigned.
+Claudie uses the concept or roles while configuring the load balancers from the input manifest. Each role represents a loadbalancer configuration for a particular use. Roles are then assigned to the load balancer cluster. A single load balancer cluster can have multiple roles assigned.
```

Comment:
`concept or roles` -> `concept of roles`

Reply:
THE KEYS ARE LIKE RIGHT NEXT TO EACH OTHER :D